Update reference card (#120)
rizinorg/book: src/refcard/intro.md

| Command                 | Description                             |
|:------------------------|:----------------------------------------|
| help                    | First introduction                      |
| s (tab)                 | Seek to a different place               |
| aa[a]                   | Auto analyze (three `a` for more)       |
| afl                     | List functions                          |
| afvl                    | List function local vars and args       |
| avg                     | List globals                            |
| t                       | List types                              |

> | iz[z]                   | List strings (two `z` for more)         |

Reviewer: this is mentioned twice.

| pdf @ [funcname](Tab)   | Disassemble function (main, fcn, etc)   |
| x [nbytes]              | Hexdump of nbytes, $b by default        |
| wx [nbytes]             | Write hexadecimal string                |
| axt @ [flag/address]    | Find cross reference to a flag/address  |

## Flags

Flags are like bookmarks, but they carry some extra information like size, tags or associated flagspace. Use `f` commands to list, set, get them.

| Command             | Description           |
|:--------------------|:----------------------|
| f name              | Add flag "name"       |
| f- name             | Remove flag "name"    |
| fl                  | List flags            |
| fd $$               | Describe an offset    |
| fN [name]           | Show the real name    |
| fx [flagname]       | Show hexdump of flag  |
| fC [name] [comment] | Set flag comment      |

## Functions and variables

> Functions appear after auto-analysis or after adding them manually, e.g. with `af` command.

Reviewer: ```suggestion
Functions appear after auto-analysis or after adding them manually using the `af` command.
```

| Command             | Description               |
|:--------------------|:--------------------------|
| af                  | Analyze function          |
| af- name            | Remove function "name"    |
| afb                 | List basic blocks         |
| afi                 | Show function information |
| afs                 | Show function signature   |

> | afvl                | Show function variables   |

Reviewer: maybe is worth mentioning agf for the function block graph

Author: Added

## Global variables

Global variables appear after auto-analysis or after adding them manually.

| Command             | Description               |
|:--------------------|:--------------------------|
| avg                 | Show all globals          |
| avg name            | Show global "name"        |
| avga name type      | Add global variable       |
| avgp name           | Print global variable     |
| avgx name           | Show xrefs to the global  |

## Information

Binary files have information stored inside the headers. The `i` command uses the RzBin API and allows us to the same things rz-bin does. Those are the most common ones.

| Command | Description                 |
|:--------|:----------------------------|
| ii      | Information on imports      |
| iI      | Info on binary              |
| ie      | Display entrypoint          |
| iS      | Display sections            |
| ir      | Display relocations         |

> | iz      | List strings (izz for more) |

Reviewer: i would split this and say that `iz` is the strings found in the read only sections meanwhile `izz` are all the strings found in the raw binary

Author: Removed this occurrence then
